### PR TITLE
adding current directory explicitly

### PR DESCRIPTION
**Pull request by einhverfr**

### CODE REVIEW

File: lsmb-request.pl [hunk 1]
```diff
@@ -19,7 +19,7 @@ =head1 COPYRIGHT
 =cut
 
 package LedgerSMB::Handler;
-
+use lib '.';
```
Comment (sbts) [inline lsmb-request.pl]: rather than adding '.' as a relative path, can we explicitly add the Current Working Directory?

Comment (sbts) [inline lsmb-request.pl]: I'd propose this block of code
```
use Cwd 'abs_path';
use File::Basename;
my $dirname = abs_path(dirname(__FILE__));
use $dirname;
```
Or something similar

Comment (sbts) [inline lsmb-request.pl]: Actually, that exact solution won't quite work if the file isn't in the TL dir of LedgerSMB, rather we would want to strip off the trailing portion of the path, so we end up with the correct parent.

Comment (einhverfr) [inline lsmb-request.pl]: you would want to use FindBin, but it is also currently broken on Debian so......

Comment (einhverfr) [inline lsmb-request.pl]: After some further testing, it looks like FindBin actually works but lib->import() doesn't which is why we saw the breakage.  I think for 1.4.32 we should replace the above with:

```
use FindBin;
use lib $FindBin::Bin;
```

**Pull request merged**